packages/relay/src/lib/errors/JsonRpcError.ts

```diff
@@ -160,5 +160,13 @@ export const predefined = {
     name: 'Value too low',
     code: -32602,
     message: 'Value below 10_000_000_000 wei which is 1 tinybar'
-  })
+  }),
+  'INVALID_CONTRACT_ADDRESS':  (address) => {
+    const message = address && address.length ? ` Expected length of 42 chars but was ${address.length}.` : ''
```

Comment:
nit:
```suggestion
    const message = address && address.length ? `Expected length of 42 chars but was ${address.length}.` : ''
```

Reply:
This prevents an empty space char at the end of the error message if `address` is null